### DISCUSSION TITLE
Modifica voto,valori

### DISCUSSION THREAD

**Discussion by sempliciniluigi89** [iisgiua/giuaschool#359] [Richieste di aiuto]
Posso chiedere in quale cartella è possibile modificare i voti con i valori e la tipologia di valutazione?Ad esempio se io voglio che 9- anzicchè valga 8.75 ,voglio che valga 8.85 oppure la tipologia invece di Pratico voglio Grafico

**Comment by trinko** [comment]
Al momento non è possibile fare questo tipo di modifiche, perché la gestione è nel codice in javascript in vari template.
Dovresti andare a modificare la funzione "formatter" presente nel template della pagina di inserimento dei voti, che gestisce l'inserimento corretto del valore. E non solo quello.
